extensions/extensions-helm/aissemble-mlflow-chart/values.yaml

```diff
@@ -1,10 +1,9 @@
 mlflow:
   image:
-    # NB: will soon be updated to public mlflow image - but waiting to upgrade across the application, so point to the
-    # 2.3.1 image that was part of aissemble 1.7.0 in the meantime. TODO in https://github.com/boozallen/aissemble/issues/226
-    registry: ghcr.io
-    repository: boozallen/aissemble-mlflow
-    tag: 1.7.0
+    registry: docker.io
+    repository: bitnami/mlflow
+    tag: 2.15.1-debian-12-r0
```

Comment:
S: Unless the community chart is using `latest` as the default tag, I highly recommend not overriding it.  And for the other fields, we should only be adding them here if we're overriding a default value so any that are just copies of the default should be removed.  If they are not copies/removed, then they need to be noted in the readme table (i.e. `tag` and `pullPolicy` need to be added.)

Reply:
The tag in the chart is referencing `2.15.1-debian-12-r0`, so I will remove this tag.